src/bots/ping/tests/handler.test.js

```diff
@@ -0,0 +1,10 @@
+import client from '../../../client';
+import handler from '../handler';
+
+describe('ping event', () => {
+  test('ping handler is called when ping event is emitted', () => {
+    const logSpy = jest.spyOn(client.logger, 'debug');
+    handler({}, { channel: { send: () => {} } });
```

Comment:
Too ugly of a hack? :)

Reply:
can you possibly move that to a variable or something that gives some semantic meaning to it? Otherwise just looking at it gives little indication what it should be and why it's sent.

Reply:
I added a comment in a commit pushed just now. Would a variable still be better?